pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/dashboard/DashboardUseCase.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dsm.model.dashboard;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import org.broadinstitute.dsm.db.dao.dashboard.DashboardDao;
+import org.broadinstitute.dsm.db.dto.dashboard.DashboardDto;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.model.elastic.search.ElasticSearchable;
+import org.elasticsearch.action.search.MultiSearchResponse;
+import org.elasticsearch.index.query.QueryBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DashboardUseCase {
+
+    private static final Logger logger = LoggerFactory.getLogger(DashboardUseCase.class);
+    private DashboardDao dashboardDao;
+    private ElasticSearchable elasticSearchable;
+
+    public DashboardUseCase(DashboardDao dashboardDao, ElasticSearchable elasticSearchable) {
+        this.dashboardDao = dashboardDao;
+        this.elasticSearchable = elasticSearchable;
+    }
+
+    public List<DashboardData> getByDdpInstance(DDPInstanceDto ddpInstanceDto) {
+        List<DashboardData> result = new ArrayList<>();
+        List<DashboardDto> dashboardDtos = dashboardDao.getByInstanceId(ddpInstanceDto.getDdpInstanceId());
+        logger.info("Collecting dashboard graphs for instance: " + ddpInstanceDto.getInstanceName());
+        for (DashboardDto dashboardDto: dashboardDtos) {
```

Comment:
what is the inside of this for loop trying to do?  please add a comment explaining intent